File: cifra-de-control/fisier.java
import java.util.Scanner;
class HelloWorld {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        if (a % 9 == 0) 
            System.out.println("Cifra de control este 9");
        else 
            System.out.println("Cifra de control este " + a % 9);
    }
}

File: cifra-de-control/fisier.cpp
#include <iostream>
using namespace std;
int main() {
    int a; cin >> a;
    if (a % 9 == 0) 
        cout << "Cifra de control este 9";
    else
        cout << "Cifra de control este " << a % 9;
    return 0;
}
